interface/forms/gad7/new.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/**
+ * gad-7 form using forms api     new.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ruth Moulton <moulton ruth@muswell.me.uk>
+ * @copyright Copyright (c) 2021 ruth moulton <ruth@muswell.me.uk>
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("gad7.inc.php"); //common strings, other includes etc
```

Comment:
add to this comment that this require_once calls does "call to globals.php" so we don't get confused in the future